src/components/QuranReader/index.tsx

```diff
@@ -30,6 +33,8 @@ type QuranReaderProps = {
   quranReaderDataType?: QuranReaderDataType;
 };
 
+const DEBOUNCE_WAIT_TIME = 2000; // 2 seconds
```

Comment:
nit: this should be renamed to `WAITING_TIME_DEBOUNCE_WAIT_TIME` since it's in the QuranReader